modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/js/header/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {openModal} from 'frontend-js-components-web';
+
+const EDIT_ERC_MODAL_ID = 'edit-erc-modal';
+
+const ERCButtonPropsTransformer = ({additionalProps, ...props}) => ({
```

Comment:
I think this should be extended (externalReferenceCode). multiple times has been required to change it